src/utils/helperFunctions.ts

```diff
@@ -0,0 +1,33 @@
+import {LabTest} from '../types/selectTest'
```

Comment:
Why is this code in a helperFunctions() method. Shouldn't it either be in LabTest object, or a Panel object, or some class that represents LabTests? (Domain object class). This code seems to be outside any domain, as just "helper". 

Thoughts?

Reply:
LabTest here is a data type which we are specifying in the function argument. The reason behind having getShortName() and getTestName() in a separate helperFunctions is so that these functions can be used across lablite where ever labtest names need to be displayed